go/chat/boxer.go

```diff
@@ -281,6 +281,7 @@ func (b *Boxer) checkInvariants(ctx context.Context, convID chat1.ConversationID
 }
 
 func (b *Boxer) unbox(ctx context.Context, boxed chat1.MessageBoxed, encryptionKey types.CryptKey) (*chat1.MessageUnboxedValid, UnboxingError) {
+	b.Debug(ctx, "Unboxing message version: %v", boxed.Version)
```

Comment:
I guess. I just put those in to have some visibility if something goes awry. Too noisy?